src/form-builder/components/FormPrivilegeTable.jsx

```diff
@@ -216,40 +207,15 @@ export default class FormPrivilegeTable extends Component {
     }
     return null;
   }
-
-  _saveFormResource(formJson) {
```

Comment:
@buvaneswari-arun why did you remove this method?

Reply:
When this method is called here, then everytime a form privilege is saved, the formResource is also saved and an entry is made in the FormResource table. Now, I have restricted this updation to take place only when the whole form is saved and not everytime a privilege is saved. When the whole form is saved, then the privileges are gathered and written in the JSON